rpc/grpc/grpc_test.go

```diff
@@ -1,9 +1,19 @@
+//nolint:dupl
```

Comment:
why disable this linter?

Reply:
I didn't spend time refactoring the testing suite so it has multiple duplicates. I'll open a good first issue for it

Reply:
https://github.com/celestiaorg/celestia-core/issues/1590